resources/codestar-connections.go

```diff
@@ -0,0 +1,74 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/codestarconnections"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type CodeStarConnection struct {
+	svc              *codestarconnections.CodeStarConnections
+	connectionArn    *string
```

Comment:
```suggestion
	connectionARN   *string
```

We try to use golang convetions wherever we can :)